pages/content/ch2/proof-patterns.mdx

```diff
@@ -0,0 +1,253 @@
+        If $G$ is valid, then $F \models G$ holds for all $F$ (that's also why we write $\models G$ if $G$ is a tautology).
+    </div>
+    <div label="hint">
+        So we need to find two formulas $F, G$ which are not tautologies, but $F \not\models G$ ...
+    </div>
+    <div label="answer">
+        There are many counterexamples, one of them is:
+        $$
+            F = A, G = A \land B
+        $$
+        ---
+
+        Both formulas are not tautologies, but $F \models G$ does not hold:
+
+        |$A$|$B$|$A$|$A \land B$|
+        |---|---|---|---|
+        |$0$|$0$|$0$|$0$|
+        |$0$|$1$|$0$|$0$|
+        |$1$|$0$|$1$|$0$|
+        |$1$|$1$|$1$|$1$|
+
+        The third row shows that $F \models G$ does not hold.
+    </div>
+</Problem>
+
+## Existence Proof
+A special type of an existence proof is the proof by Counterexample; to show that $F$ is not true, we show that "not $F$" is true for some input.
+
+## Proof by Contradiction
+
+<Problem title="Problem 1" source="Yannick Funke">
+    <div label="question">
+        Prove the following statement:
+        $$
+            x > 0 \land y > 0 \Longrightarrow \sqrt{x + y} \ne \sqrt{x} + \sqrt{y}
+        $$
+        Prove it using a proof by contradiction.
+    </div>
+    <div label="hint">
+        Try to prove $x > 0 \land y > 0 \Longrightarrow \sqrt{x + y} = \sqrt{x} + \sqrt{y}$ and show that this leads to a contradiction; then choose $T$.
+    </div>
+    <div label="answer">
+        Statement $T$:
+        $$
+            x > 0 \land y > 0 \land (x = 0 \lor y = 0) 
+        $$
+        Statement $T$ is obviously false:
+        $$
+            \begin{aligned}
+                & x > 0 \land y > 0 \land (x = 0 \lor y = 0) \\
+                \dot\iff & \left[ (x > 0 \land x = 0) \lor (x > 0 \land y = 0) \right] \land (y > 0) && \text{Distributive Law} \\
+                \dot\iff & \left[ \bot \lor (x > 0 \land y = 0) \right] \land (y > 0) && x > 0 \land x = 0 \equiv \bot \\
+                \dot\iff & (x > 0 \land y = 0) \land (y > 0) && \bot \lor F \equiv F \\
+                \dot\iff & x > 0 \land (y = 0 \land y > 0) && \text{Associative Law} \\
+                \dot\iff & x > 0 \land \bot && y = 0 \land y > 0 \equiv \bot \\
+                \dot\iff & \bot && F \land \bot \equiv \bot
+            \end{aligned}
+        $$
+
+        Assume that $S$ is false, i.e. $x > 0 \land y > 0 \land \sqrt{x + y} = \sqrt{x} + \sqrt{y}$ holds. Then:
+        $$
+            \begin{aligned}
+                &&x > 0 \land y > 0 \land \sqrt{x + y} &= \sqrt{x} + \sqrt{y} \\
+                &\dot\implies& x > 0 \land y > 0 \land x + y &= x + 2\sqrt{xy} + y \\
+                &\dot\implies& x > 0 \land y > 0 \land 0 &= 2\sqrt{xy} \\
+                &\dot\implies& x > 0 \land y > 0 \land 0 &= \sqrt{xy} \\
+                &\dot\implies& x > 0 \land y > 0 \land 0 &= xy \\
+                &\dot\implies& x > 0 \land y > 0 \land (x = 0 &\lor y = 0)
+            \end{aligned}
```

Comment:
Increase visibility using `alignat` to align in two columns.

Reply:
inwiefern? `\begin{alignat}{3}`? dann gibt es so ein numbering ... und weniger space ...